lib/deck.rb

```diff
@@ -0,0 +1,17 @@
+class Deck 
+    attr_reader :cards
+    
+    def initialize(cards) 
+        @cards = cards
+    end
+
+    def count
+        cards.count
+    end
```

Comment:
In this method, you're calling to `cards` instead of `@cards`. This works, because you're calling to the attr_reader on line 2, which returns the instance variable on line 5. Since you have access to the instance variable, call to that rather than the attr_reader. The purpose of the attr reader is so that you can call to that attribute when you're outside of this class.